v23.1/query-behavior-troubleshooting.md

```diff
@@ -21,6 +21,8 @@ Such long-running queries can hold intents open for (practically) unlimited dura
 
 Refer to the performance tuning recipe for [identifying and unblocking a waiting transaction](performance-recipes.html#waiting-transaction).
 
+If you experience this issue on a {{ site.data.products.serverless }} cluster, the cause could be [throttled Request Units](../cockroachcloud/troubleshooting-page.html#hanging-or-stuck-queries).
```

Comment:
```suggestion
If you experience this issue on a {{ site.data.products.serverless }} cluster, your cluster may be throttled or disabled because you've reached your monthly [resource limits](../cockroachcloud/troubleshooting-page.html#hanging-or-stuck-queries).
```

Reply:
included suggestion